spec.bs

```diff
@@ -189,6 +189,12 @@ example flows and do not limit the scope of how the API can be used.
 This is the first-time flow, in which a new credential is created and stored by
 an issuing bank.
 
+**TODO**: Suggest we add an extra example, which might be more widely used: When a user navigates to bank.com for internet banking and 1) their account access 
```

Comment:
I'm not opposed to such an example, though from an ideological view I continue to think we shouldn't let historical baggage of complex activations drag down WebAuthn/SPC credential creation :).

There is a slight complication: currently we are unsure whether the WebAuthn WG will allow an SPC credential to be entirely equivalent to a WebAuthn credential (i.e. make them interchangeable), or if they will require an SPC credential to have a special bit so that the RP knows this credential can be used in a third-party auth ceremony.

* If a special bit is required, this example will make a lot of sense. 
* If a special bit is **not** required, this example will just describe normal WebAuthn credential creation and thus be rather redundant imo.

Reply:
If we're aligning to Webauthn, then I would suggest we remove the detailed explanation and flows. It would be better to rather just refer to the WebAuthn document and process for Enrollment.

We would then just
* give high level explanation on the use-cases (the two mentioned)
* Re-iterate that this is standard WebAuthn and there is no cross domain capability in registration, so the standard WebAuthn process will be followed with no UI.

Reply:
Hi @goosth, I think there is a cross-origin iframe registration capability in SPC, without special UI in the current Chrome implementation.

Reply:
Agreed. But that is a WebAuthn standard. You still need the iframe to be opened in the RP's domain and the iframe to be given the correct permissions by the merchant. 
It's not using the SPC superpower of being called by the merchant and creating a key in the RP's domain.